packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -263,6 +263,26 @@ void main() {
         expect(scrollView.controller, primary2);
       });
 
+      testWidgets('Use custom ScrollController when scrollController is set', (WidgetTester tester) async {
+        final ScrollController customController = ScrollController();
+        final Widget reorderableList = ReorderableListView(
```

Comment:
It's probably okay to compose the entire widget tree here like this, since `reorderableList` is not being reused in any way:
```dart
await tester.pumpWidget(
  MaterialApp(
    home: ReorderableListView(
      scrollController: customController,
      onReorder: (int oldIndex, int newIndex) {},
      children: const <Widget>[
        SizedBox(width: 100.0, height: 100.0, child: Text('C'), key: Key('C')),
        SizedBox(width: 100.0, height: 100.0, child: Text('B'), key: Key('B')),
        SizedBox(width: 100.0, height: 100.0, child: Text('A'), key: Key('A')),
      ],
    );
  ),
);
```